src/internal/components/dropdown/dropdown-fit-handler.ts

```diff
@@ -121,6 +129,69 @@ export const getInteriorAvailableSpace = (
   );
 };
 
+export const getWidths = ({
+  triggerElement,
+  dropdownElement,
+  desiredMinWidth,
+  stretchBeyondTriggerWidth = false,
+}: {
+  triggerElement: HTMLElement;
+  dropdownElement: HTMLElement;
+  desiredMinWidth?: number;
+  stretchBeyondTriggerWidth?: boolean;
+}) => {
+  // Determine the width of the trigger
+  const triggerWidth = triggerElement.getBoundingClientRect().width;
+  // Minimum width is determined by either an explicit number (desiredMinWidth) or the trigger width
+  const minWidth = desiredMinWidth ? Math.min(triggerWidth, desiredMinWidth) : triggerWidth;
+  // If stretchBeyondTriggerWidth is true, the maximum width is the XS breakpoint (465px) or the trigger width (if bigger).
+  const maxWidth = stretchBeyondTriggerWidth ? Math.max(defaultMaxDropdownWidth, triggerWidth) : Number.MAX_VALUE;
```

Comment:
why it defaults to MAX_VALUE and not undefined?

Reply:
I didn't change this (only moved it around), but would it change anything? Since in line 152 we are doing a `Math.min` between this and another value which at that point should be defined. So I think the result should be the same.

Reply:
got it 👍